src/index.ts

```diff
@@ -29,6 +29,7 @@ export class Iterable<T> implements LibIterable<T> {
     }
```

Comment:
change line 22 to `if (typeof source[Symbol.iterator] !== 'function')` to be more explicit